src/Microsoft.OData.Edm/ExtensionMethods/DateAndTimeExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="DateTimeExtensions.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.OData.Edm
+{
+    public static class DateAndTimeExtensions

```

Comment:
Given that the methods here are public, if the parameters can be null, you should add check for that and throw an `ArgumentNullException`